Missing return statement for v8-M header

```
core_armv8mml.h:2549:1: error: control reaches end of non-void function [-Werror=return-type]
```
[`TZ_NVIC_GetPendingIRQ_NS` is missing](https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/Core/Include/core_armv8mml.h#L2543-L2549)
```c
  else
  {
    return(0U);
  }
```

cc @ReinhardKeil

--- a/CMSIS/Core/Include/core_armv8mml.h
+++ b/CMSIS/Core/Include/core_armv8mml.h
@@ -2546,6 +2546,10 @@ __STATIC_INLINE uint32_t TZ_NVIC_GetPendingIRQ_NS(IRQn_Type IRQn)
   {
     return((uint32_t)(((NVIC_NS->ISPR[(((uint32_t)(int32_t)IRQn) >> 5UL)] & (1UL << (((uint32_t)(int32_t)IRQn) & 0x1FUL))) != 0UL) ? 1UL : 0UL));
   }
+  else
+  {
+    return(0U);
+  }
 }
 
 

